### ISSUE TITLE
Digitizer - Datepicker only allows type date

### ISSUE THREAD

**Issue by CharlyUnicorn** [mapbender/mapbender#535]
You can't save a date with the datepicker in the digitizer element, if you use an other format than date in the table. The digitizer saves the date(now) from the moment you save the point and not the date you select in the calendar.

You can test this by adding a charvar field in your demo-digitizer table and alter the field in the yaml (date_favorite > date_test). Here is the SQL for the colum: 
 
`ALTER TABLE public.poi ADD COLUMN date_test character varying;`

This happens in the version 3.0.6 - in the version 3.0.5.3 you could save correctly with type charvar.

**Comment by AxxL** [comment]
Digitizer documentation always assumed, that the date field is a date, not a char, so I don't know if that was a feature or not, it was not as described.

3.0.6 uses Digitizer 1.1 and 3.0.5 uses Digitizer 1.0.

Any comments by @eSlider @LazerTiberius ?

**Comment by LazerTiberius** [comment]
>Digitizer documentation always assumed, that the date field is a date, not a char, so I don't know if that was a feature or not, it was not as described.
The front end date format should not be dependent on the database column format. This seems to be a bug.  And nevertheless like @CharlyUnicorn described the wrong date is saved. This should never happen

**Comment by AxxL** [comment]
@LazerTiberius Canufixit.mapbender.org? ;)

**Comment by LazerTiberius** [comment]
Does the fix has to be compatible with 3.0.5X?

**Comment by AxxL** [comment]
@CharlyUnicorn Sorry, we have to push it to the next release. It couldn't be done in time and we need/want to release a version. When it's done and we have some other tickets fixed it will be available in the next release.

I tagged it for 3.0.8, could be also a bugfix release with another milestone.

**Comment by AxxL** [comment]
Fixed in Digitizer what will become version 1.2.

```
select date_favorite, date_test from poi;

date_favorite |date_test  |
--------------|-----------|
2018-07-30    |30-07-2018 |
2017-12-28    |28-12-2017 |
2010-12-24    |24-12-2010 |


select column_name, data_type from information_schema.columns where table_name = 'poi';

column_name       |data_type         |
------------------|------------------|
gid               |integer           |
name              |character varying |
type              |character varying |
abstract          |character varying |
public            |boolean           |
date_favorite     |date              |
date_test         |character varying |
